aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/BaseHandlerStd.java

```diff
@@ -196,10 +196,11 @@
                 .collect(Collectors.toSet());
 
         if (!invalidParameters.isEmpty()) {
-            return ProgressEvent.defaultFailureHandler(
-                    new CfnInvalidRequestException("Invalid / unmodifiable / Unsupported DB Parameter: " + invalidParameters.stream().findFirst().get()),
-                    HandlerErrorCode.InvalidRequest
-            );
+            return ProgressEvent.failed(
+                    progress.getResourceModel(),
+                    progress.getCallbackContext(),
+                    HandlerErrorCode.InvalidRequest,
+                    "Invalid / unmodifiable / Unsupported DB Parameter: " + invalidParameters.stream().findFirst().get());
```

Comment:
nit: could you please uniform the word case spelling?

Reply:
Done.